### PR TITLE
Simplify image building

### PR DESCRIPTION
**Pull request by greg-agacinski**

### CODE REVIEW

File: codefresh.yml [hunk 1]
```diff
@@ -15,16 +15,13 @@ steps:
...
 
-  install:
-    title: "Install sloctl"
-    type: "freestyle" # Run any command
-    image: "ubuntu:latest" # The image in which command will be executed
-    working_directory: "${{clone}}" # Running command where code cloned
+  version:
+    title: "Get sloctl version"
+    type: "freestyle"
+    image: 'quay.io/codefreshplugins/alpine:3.8'
```
Comment (nieomylnieja) [inline codefresh.yml:21]: this is an old alpine version, any reason why we're using it here? I know it's just for that little wget jq combo, just curious

**Review by nieomylnieja — CHANGES_REQUESTED** [review]
nicely done! This is much better now

File: codefresh.yml (hunk 1 above)
Comment (greg-agacinski) [reply codefresh.yml:21]: I've just used the alpine version provided by codefresh. I'll upgrade to the newest version served from docker hub

Comment (nieomylnieja) [reply codefresh.yml:21]: sure, I didn't mean it's anything bad to use this one here, was just wondering if there was a deeper meaning here :P

Comment (greg-agacinski) [inline codefresh.yml:21]:
```suggestion
    image: 'alpine:3.17.2'
```

**Review by nieomylnieja — APPROVED** [review]

**Pull request merged**